src/struct_mv/struct_communication.c

```diff
@@ -659,7 +665,9 @@ hypre_CommPkgCreate( hypre_CommInfo   *comm_info,
 }
 
 /*--------------------------------------------------------------------------
- * Note that this routine uses an identity coordinate transform
+ * hypre_CommBlockSetEntries
```

Comment:
Note that I have been removing the extra function name in the comments because it's redundant and I've found it to be incorrect at times.  So, we are working against each other with this. :)  Not a big deal, but I thought I'd mention it.

Reply:
Thanks for mentioning! I wasn't aware of your effort. I will refrain from adding comment entries like this